src/renderer/template/src/core/colors/variables.before.scss

```diff
@@ -2,11 +2,66 @@
 $color-hover-lighten-amount: 4 !default;
 $color-active-darken-amount: $color-hover-lighten-amount !default;
 $color-border-darken-amount: $color-active-darken-amount * 4 !default;
+$color-border-lighten-amount: $color-border-darken-amount !default;
 
-$color-white: #fdfdfd;
-$color-white-text: #1f1f1f;
+$color-white: #181818;
+$color-white-text: #dcdcdc;
 
-$color-base: #fdfdfd;
-$color-base-text: #1f1f1f;
+$color-base: #181818;
+$color-base-text: #dcdcdc;
 
-$color-black: #1f1f1f;
+$color-black: #dcdcdc;
+
+$color-primary: #4f8cc9 !default; // GitHub-Dark
+$color-secondary: #ef6c00 !default; // Orange-800
+
+$color-gray: #e0e0e0 !default; // Gray-300
+$color-green: #4caf50 !default; // Green-500
+$color-orange: #ff9800 !default; // Orange-500
+$color-red: #f44336 !default; // Red-500
+$color-yellow: #fabf40 !default; // Yellow-700
+
+/* PRIMARY */
+
+$color-primary-bg: $color-primary !default;
+$color-primary-border: lighten( $color-primary-bg, $color-border-lighten-amount ) !default;
```

Comment:
How do I lighten instead of darken? Original `variables.css` has `darken` function in border color calculation and it doesn't accept negative values.
I had to manually import and brighten every color using `lighten( $color-primary-bg, $color-border-lighten-amount )`.